wordle-web/components/keyboard.vue

```diff
@@ -1,9 +1,10 @@
 <template>
-  <v-card class="my-5 pa-5">
-    <v-row v-for="(charRow, i) in chars" :key="i" no-gutters justify="center">
-      <v-col v-for="char in charRow" :key="char" cols="1">
-        <v-container class="text-center">
+  <v-card class="ma-0 pa-0" color="transparent" flat>
```

Comment:
There is supposed to be a gradient background here for the game board. try using `style="gradient code here"` as part of the v-card